config.ts

```diff
@@ -61,6 +61,7 @@ interface WebsiteConfig {
   discord: string;
   instagram: string;
   linkedin: string;
+  hackathon_url: string;
```

Comment:
Can we move this above all the social links its not really a social link its more of a configuraiton. maybe this shouldnt be in `website_config` but under like a `main_event` section at the top. that we can change whenever we need to build custom integrations like this.